spec.html

```diff
@@ -44450,6 +44435,22 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-issamebuffer" type="abstract operation">
+        <h1>
+          IsSameBuffer (
+            _a_: an ArrayBuffer or SharedArrayBuffer,
+            _b_: an ArrayBuffer or SharedArrayBuffer,
+          ): a Boolean
+        </h1>
+        <dl class="header">
+        </dl>
+        <emu-alg>
+          1. If SameValue(_a_, _b_) is *true*, return *true*.
+          1. If IsSharedArrayBuffer(_a_) is *true*, IsSharedArrayBuffer(_b_) is *true*, and _b_.[[ArrayBufferData]] is _a_.[[ArrayBufferData]], return *true*.
+          1. Return *false*.
```

Comment:
Can't this technically be simplified to

```suggestion
          1. If _a_.[[ArrayBufferData]] is _b_.[[ArrayBufferData]], return *true*.
          1. Return *false*.
```

?

Reply:
This IsSameBuffer algorithm is derived from the current step 18 of [SetTypedArrayFromTypedArray](https://tc39.es/ecma262/multipage/indexed-collections.html#sec-settypedarrayfromtypedarray). Your proposed change is editorial unless a downstream spec is allowing distinct ArrayBuffer instances to be backed by the same Data Block, for which it would alter output from **false** to **true** (but probably with positive results). If we're confident about that, however, I'd probably just inline all the uses rather than introducing a new operation.

Reply:
I would've assumed it would always be the case that distinct ArrayBuffers are backed by distinct Data Blocks. Inlining my suggestion sounds fine.

Reply:
Done.